### DISCUSSION TITLE
Option to Customize or Hide Icons on the Right Side of the URL Bar to Reduce Clutter

### DISCUSSION THREAD

**Discussion by gumbarros** [zen-browser/desktop#3868] [Ideas]
### **Is your feature request related to a problem? Please describe.**
The right side of the URL bar contains multiple icons such as site-specific buttons, extensions, or menu items. While some of these are useful, the area can become cluttered and distracting, especially when too many items are present. It can also interfere with accessing core browser functionality.

### **Describe the solution you'd like**
I would like an option to hide or customize which items appear on the right side of the URL bar. This could be implemented as:
1. A toggle in settings to hide all optional items in the right side of the URL bar.
2. An option to selectively enable/disable certain icons or buttons.

This would give users more control over the appearance of the URL bar and reduce unnecessary clutter.

### **Describe alternatives you've considered**
1. Using a Zen Mods
2. Can someone send a CSS snippet that only shows the favorite button?

### **Additional context**
This feature would help streamline the browsing experience, particularly for users who value minimalism, have small screens or are using the single toolbar that have smaller space. Reducing visual clutter enhances focus and usability.

**Comment by gumbarros** [comment]
Accidentally opened a issue at [NCalc](https://www.github.com/ncalc/ncalc) instead of here, thats why I'm following an issue template lol.

**Comment by YouriCode** [comment]
I agree, sometimes I cannot even access my URL bar because of the buttons that appear on hover.
A solution to this could be to have these buttons appear UNDER the URL on hover. The bar’s height would expand with the same animation and we could access all these buttons while hovering the URL bar, and still be able to click the address.

How it is currently:
![URL bar](https://github.com/user-attachments/assets/4df4cb64-52ed-4082-8eff-db89a5a5085c)
![URL bar on hover](https://github.com/user-attachments/assets/194975bc-11a6-45c9-95bf-7ee1ebe5a6c2)

My proposition:
![Proposition](https://github.com/user-attachments/assets/e1cc8628-b84a-48c6-870f-113649375bd3)
And we have to imagine there’s no second lock icon underneath the URL...
